.ebextensions/privatekey.config

```diff
@@ -0,0 +1,20 @@
+Resources:
+  AWSEBAutoScalingGroup:
+    Metadata:
+      AWS::CloudFormation::Authentication:
+        S3Auth:
+          type: "s3"
+          buckets: ["elasticbeanstalk-us-east-1-115630815585"]
+          roleName: 
+            "Fn::GetOptionSetting": 
+              Namespace: "aws:autoscaling:launchconfiguration"
+              OptionName: "IamInstanceProfile"
+              DefaultValue: "aws-elasticbeanstalk-ec2-role"
+files:
+  # Private key
+  "/mu-auth-73f4d05ed60d.json":
+    mode: "000400"
+    owner: root
+    group: root
+    authentication: "S3Auth"
+    source: https://elasticbeanstalk-us-east-1-115630815585.s3.amazonaws.com/privatekey/mu-auth-73f4d05ed60d.json
```

Comment:
should this url be a string?

Reply:
https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/https-storingprivatekeys.html Not per the tutorial